mix.exs

```diff
@@ -10,8 +10,8 @@ defmodule Uploadex.MixProject do
       deps: deps(),
       elixirc_paths: elixirc_paths(Mix.env()),
       name: "Uploadex",
-      source_url: "https://github.com/gabrielpra1/uploadex",
-      description: "Elixir library for handling uploads using Ecto and Arc",
+      description: "Elixir library for handling uploads using Ecto",
```

Comment:
good catch!

But maybe using the github's description? "Elixir library for handling uploads with Ecto, Phoenix and Absinthe"

Reply:
That's better!